src/main/java/ro/msg/learning/shop/DummyData.java

```diff
@@ -0,0 +1,110 @@
+package ro.msg.learning.shop;
+
+import ro.msg.learning.shop.model.Address;
+import ro.msg.learning.shop.model.Customer;
+import ro.msg.learning.shop.model.Location;
+import ro.msg.learning.shop.model.Product;
+import ro.msg.learning.shop.model.ProductCategory;
+import ro.msg.learning.shop.model.Stock;
+import ro.msg.learning.shop.model.Supplier;
+
+import java.math.BigDecimal;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public final class DummyData {
+    public static final Address address1;
```

Comment:
Please use a suggestive naming on the dummy objects (address, products, location etc)

Reply:
I renamed address1 to address. As for the products, I created two of them, but I think the best way to differentiate between them is to append a number to their names (e.g. product1, product2 -> two different products, their fields are irrelevant). As for the locations I defined "location20" to be a location having all products in stock, each with a quantity of 20. I renamed it to "locationWithStock20" for better readability.

Reply:
renamed them